packages/orchestrator-ui-components/src/components/WFOTable/utils/withSortableAndFilterableValues.ts

```diff
@@ -0,0 +1,23 @@
+import { WFOTableColumns, WFOTableDataColumnConfig } from './columns';
+
+export function withSortableAndFilterableValues<T>(
```

Comment:
I'm not sure about this name. This naming is used by other libraries when they create an Higher Order Component which this one is not. No good alternative come to mind though

Reply:
Changing it to mapSortableAndFilterableValuesToTableColumnConfig